### PR TITLE
fix #78369 vip.de

### LINKED ISSUE #78369: www.vip.de
### Issue URL (Annoyance)
 [https://www.vip.de/cms/prinz-philip-wieder-fit-die-queen-schickt-blumen-ans-krankenhaus-4728851.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.vip.de%2Fcms%2Fprinz-philip-wieder-fit-die-queen-schickt-blumen-ans-krankenhaus-4728851.html)
### Comment
Username: @Morku
> Cookie Pop-up
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German

### PR DESCRIPTION
**Pull request by wout-codes**
#78369

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -86,7 +86,7 @@ macworld.co.uk,welt.de,desired.de,sportbuzzer.de,fitbook.de,t-online.de,wunderwe
...
+/wrapperMessagingWithoutDetection.js$script,domain=sportbible.com|vox.de|essen-und-trinken.de|jameda.de|gutefrage.net|planetiphone.de|computerfrage.net|helpster.de|heise.de|dpd.com|dpd.de|duden.de|sky.de|sky.com|tvmovie.de|cardscout.de|meineorte.com|tecchannel.de|itpro.co.uk|tvnow.at|tvnow.de|netdoktor.de|computerwoche.de|the-sun.com|thescottishsun.co.uk|thesun.co.uk|thesun.ie|handelsblatt.com|macwelt.de|lancashiretelegraph.co.uk|idealo.de|lvz.de|myhomebook.de|travelbook.de|4players.de|n-page.de|planet3ds.de|planetds.de|planetgameboy.de|planetswitch.de|planetvita.de|playstationportable.de|portablegaming.de|gala.de|denofgeek.com|classicfm.com|globalplayer.com|radio.pl|politico.eu|expertreviews.co.uk|radio.net|rtlplus.de|rtl-passion.de|geo-television.de|gruenderkueche.de|techstage.de|praxisvita.de|vip.de
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:89]: You break their article - twitter related posts aren't loading.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -39,6 +39,7 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
...
 !
 dpchas.com.ua##.social-sharing-buttons__button
+vip.de##.rtli-master-mt-social-embed
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:42]: Twitter related posts in the article shouldn't be blocked.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — CHANGES_REQUESTED** [review]
Add `vip.de` to the existing rule `businessinsider.de#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var g = new MutationObserver(function() {var b = document.querySelector('button[title^="Alle akzeptieren"]');var c = document.querySelector('html');b && b.click();c && c.style.setProperty('overflow', 'visible', 'important');c && c.style.setProperty('position', 'static', 'important');});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});`

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: Okay, wait. Lemme check.

**Review by zloyden — APPROVED** [review]

**Pull request merged**